data/core/chitchat.md

```diff
@@ -217,12 +217,13 @@
 * affirm
     - utter_ask_migration
 * deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
-* affirm OR how_to_get_started{"product":"stack"}
+    - utter_explain_rasa_components
+    - utter_rasa_components_details
+    - utter_explain_nlucorex
+* affirm OR how_to_get_started{"product":"all"}
```

Comment:
I agree, but "Would you like me to tell you more about ____" sounds ambiguous. Is it expecting a yes/no or a specification of what the user wants to know about. I guess it's a smart way of combining the two into one, but I thought it could use some clarity on what it's trying to be.

Reply:
> Is it expecting a yes/no or a specification of what the user wants to know about.

This is exactly why we have `* affirm OR how_to_get_started{"product":"all"}` 😉